mujmap.toml.example

```diff
@@ -58,6 +58,16 @@ password_command = "pass example@fastmail.com"
 
 # cache_dir =
 
+## The location of the mail dir, where downloaded email is finally stored. If
+## not given, mujmap will try to figure out what you want. You probably don't
+## want to set this.
```

Comment:
We should be more exact here about what "try to figure out what you want" means. You've already written a description of how this works more or less in the PR description, so we can repeat this here.

Eventually I want to bring the documentation out of this file and into a proper manpage or something.

Reply:
Alright, I've fleshed this out a bit. I'm not really happy with it but I couldn't see a better way without introducing some concepts first.